### PR TITLE
config: modify commit message template for ti-community-infra

### PR DESCRIPTION
**Pull request by Mini256**

### CODE REVIEW

File: prow/config/config.yaml [hunk 1]
```diff
@@ -323,6 +323,9 @@ tide:
     pingcap/tidb: squash
 
   merge_commit_template:
+    ti-community-infra/tichi:
```
Comment (hi-rustin) [inline prow/config/config.yaml:326]: Can you help to see if this supports using org names for all repository configurations? Not just tichi.

**Review by hi-rustin — CHANGES_REQUESTED** [review]

File: prow/config/config.yaml (hunk 1 above)
Comment (Mini256) [reply prow/config/config.yaml:326]: Support

image

Comment (hi-rustin) [reply prow/config/config.yaml:326]: Cool

**Review by hi-rustin — APPROVED** [review]

**Pull request merged**